### DISCUSSION TITLE
Suggestion: Switch light/dark based on clock time

### DISCUSSION THREAD

**Discussion by ldeluigi** [community/community#22031] [Accessibility]
<p>Please add the possibility to turn dark mode on automatically when the client is between some time representing actual night, or make start/stop configurable.</p>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/ldeluigi">@ldeluigi</a>, thanks for sharing this feedback! If in your OS settings you have your appearance set to be Auto, you can select ‘Default to system’ in GitHub to automatically have GitHub transition between light and dark modes at the local time of your sunrise or sunset. Would that solve what you are looking to achieve?</p>

**Comment by haoranpb** [comment]
<p>Hi <a class="mention" href="/gaknoll">@gaknoll</a>. Yes, your suggestion could solve the problem for some people. But for me, my OS setting is always in dark mode. Because for the OS, I feel like the dark mode is overall a better design, so your solution would not work for me and this feature request would be very helpful to people like me.</p>
<p>Finally, the dark theme is really a great feature, thank you for your work!</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Got it, so an OS independent Auto mode for GitHub. Thanks for chiming in <a class="mention" href="/ludanxer">@ludanxer</a>!<br>
<a class="mention" href="/ldeluigi">@ldeluigi</a> does that also align with the scenario you were envisioning?</p>

**Comment by ldeluigi** [comment]
<p>Absolutely. Thank you very much!</p>

**Comment by gaknoll** [comment]
<p>2 posts were merged into an existing topic: <a href="https://github.community/t/ability-to-quickly-toggle-light-and-dark-mode/147821/6">Ability to quickly toggle light and dark mode</a></p>

**Comment by Hades32** [comment]
<p>Just like my phone switches to dark mode in apps after sun-set, I want the same behavior in GitHub.</p>

**Comment by gaknoll** [comment]
<p><a class="mention" href="/hades32">@Hades32</a> Right now this can be achieved by setting your OS theme settings to <code>Auto</code> and subsequently setting your GitHub theme to <code>Default to system</code> which would allow GitHub to automatically switch to dark mode on sunset.<br>
Are you alternatively requesting an OS independent <code>Auto</code> setting in GitHub?</p>

**Comment by Hades32** [comment]
<p>AFAIK there is no “auto” for dark-mode on Windows 10. (Only for “night mode” which is something else).</p>
<p>Although I see that this is a shortcoming of windows, I think there is a legitmate use-case to not have the OS in dark-mode, but still wanting to change websites that are very bright into a dark-mode. So, yes, I’d ask for and OS independent auto/sun-set&amp;rise setting.</p>

**Comment by szekelyisz** [comment]
<p>Yes please! KDE user here, which also lacks auto-dark-mode but I personally like my desktop to stay dark and websites switch automatically according to the current time. Now I’m using a browser extension to switch between themes but I have to do it manually twice every day.</p>

**Comment by underhandedness** [comment]
<p>Time of day best as suggested by bluelight filter people as opposed to sundown locally.</p>

**Comment by blakegearin** [comment]
<p>+1 I would like this feature as well. Ideally a user would be able to choose the light and dark hours.</p>

**Comment by cehfisher** [comment]
Hi @ldeluigi - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/22031#discussioncomment-3235220). Could you please confirm this resolves your issue and close out this request? Thank you! (cc @queenofcorgis)
